docker/Dockerfile

```diff
@@ -176,6 +176,6 @@ ENV GLOBAL_AGENT_ENVIRONMENT_VARIABLE_NAMESPACE=''
 # The `--no-node-snapshot` node option enables the usage of the backstage scaffolder on nodejs 20
 # https://github.com/backstage/backstage/issues/20661
 
-ENTRYPOINT ["node", "packages/backend", "--no-node-snapshot", "--config", "app-config.yaml", "--config", "app-config.example.yaml", "--config", "app-config.example.production.yaml"]
+ENTRYPOINT ["node", "--require", "./instrumentation.ts", "packages/backend", "--no-node-snapshot", "--config", "app-config.yaml", "--config", "app-config.example.yaml", "--config", "app-config.example.production.yaml"]
```

Comment:
I think the instrumentation.js file needs to be copied from source to container as noted in the upstream [docs](https://github.com/backstage/backstage/blob/0d5d4215ac444b0dc51689ebe246e14de44fdc52/docs/tutorials/setup-opentelemetry.md#production-setup): 
`COPY --chown=${NOT_ROOT_USER}:${NOT_ROOT_USER} packages/backend/src/instrumentation.js ./` 

Consider inserting after this [line](https://github.com/janus-idp/backstage-showcase/blob/main/docker/Dockerfile#L61)?

Reply:
Yes ty, added!